meagea-api/src/test/java/project/PromotionServiceTest.java

```diff
@@ -151,13 +150,31 @@ public void updatePromotionSuccessTest() throws IOException {
     public void updatePromotionFailTest() throws IOException {
         List<MultipartFile> list = new ArrayList<>();
         for (int i = 0; i < 4; i++) {
-            File file = new File("src\\main\\java\\project\\image\\" + "file" + i + ".jpg");
+            File file = new File("/Users/gim-eunjeong/IdeaProjects/meagea/meagea-api/src/main/java/project/image/"
+                    + "file" + i + ".jpg");
             MockMultipartFile mul = new MockMultipartFile("file" + i, new FileInputStream(file));
             list.add(mul);
         }
-        PromotionModifyDto dto = new PromotionModifyDto(1, "수정된 제목", list, "수정된 설명", "수정된 조건");
+        PromotionModifyForm dto = new PromotionModifyForm(1, "수정된 제목", list, "수정된 설명", "수정된 조건");
 
         Throwable ex = Assertions.assertThrows(NullPointerException.class, () -> service.updatePromotion(dto));
-        assertThat(ex.getMessage()).isEqualTo("수정 가능한 Promotion 객체가 존재하지 않습니다.");
+        assertThat(ex.getMessage()).isEqualTo("수정 가능한 Promotion 데이터가 존재하지 않습니다.");
+    }
+
+    @Test
+    public void saveImageFileTest() throws IOException {
+        List<MultipartFile> list = new ArrayList<>();
+        for (int i = 0; i < 4; i++) {
+            FileInputStream input = new FileInputStream("/Users/gim-eunjeong/IdeaProjects/meagea/meagea-api/src/main/java/project/image/file" + i + ".jpg");
```

Comment:
```suggestion
            File imageFile = ResourceUtils.getFile("classpath:file" + i + ".jpg");
            FileInputStream input = new FileInputStream(imageFile);
```
이런 식으로 할 수 있지 않을까 싶네요.